content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -224,7 +224,7 @@ reference additional implementation-specific configuration for this class.
 
 #### Namespace-scoped parameters
 
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
 
 `Parameters` field has a `scope` and `namespace` field that can be used to
```

Comment:
For graduation to beta, it'd be good to expand this paragraph with some context about why you might want the params for an IngressClass to be namespaced rather than cluster scoped.

Reply:
(one example: to restrict which namespaces are allowed to use that ingress type).

Reply:
I elaborated to include benefits of namespace-scoped resources. Please take another look, thank you.